client/blocks/author-selector/switcher-shell.jsx

```diff
@@ -16,19 +16,16 @@ import Popover from 'calypso/components/popover';
 import PopoverMenuItem from 'calypso/components/popover/menu-item';
 import UserItem from 'calypso/components/user';
 import InfiniteList from 'calypso/components/infinite-list';
-import { fetchUsers } from 'calypso/lib/users/actions';
 import { hasTouch } from 'calypso/lib/touch-detect';
 
 /**
  * Module variables
  */
 const debug = debugModule( 'calypso:author-selector' );
-let instance = 0;
 
-class AuthorSwitcherShell extends React.Component {
+class AuthorSwitcherShell extends Component {
 	static propTypes = {
 		users: PropTypes.array,
-		fetchingUsers: PropTypes.bool,
 		numUsersFetched: PropTypes.number,
 		totalUsers: PropTypes.number,
 		usersCurrentOffset: PropTypes.number,
```

Comment:
The component doesn't use any of `numUsersFetched`, `totalUsers` and `usersCurrentOffset`.

Reply:
fixed in https://github.com/Automattic/wp-calypso/pull/51879/commits/7baf24118edde4f5602fdc1744573cb6d09fb69b